Basis/Operators.swift

```diff
@@ -44,9 +44,21 @@ infix operator <| {
 ///		
 /// can also be written as:
 ///
-///		nubBy(==)(x) |> zip(y)
-///					 |> map(==)
-///					 |> and
+///		nubBy(==)(x) |>> zip(y)
+///					 |>> map(==)
+///					 |>> and
+infix operator |>> {
+	precedence 100
+	associativity left
+}
+
+/// Cons | Constructs a list by appending a given element to the front of a list.
+infix operator <| {
+	precedence 100
+	associativity right
+}
+
+/// Snoc | Constructs a list by appending a given element to the end of a list.
```

Comment:
This should be fine.  The occasional paretheses aren't a dealbreaker for most people, just so long as the precedence is in the right ballpark.

And yes, if you think about it, snoc can *only* work `infixl`.  If you were to try to use it `infixr`, you'd get type errors all over the place!